src/open_inwoner/components/templates/components/Card/RenderCard.html

```diff
@@ -13,13 +13,13 @@
     <div class="card__body{% if compact %}--compact{% endif %} {% if direction %} card__body--direction-{{ direction }}{% endif %}{% if grid %} card__body--grid{% endif %}">
         {% if title %}
             {% if compact %}
-            <h3 class="utrecht-heading-3 card__heading-3">
+            <p class="utrecht-heading-3 card__heading-3">
```

Comment:
Why are we switching to `p` for the card title? It would seem that some `h` element (like the `h3` that we currently) is appropriate (especially since we keep the corresponding style)?